requirements.d/development.txt

```diff
@@ -1,11 +1,11 @@
-setuptools !=60.6.0, !=60.7.0
-setuptools_scm
-pip
-virtualenv
-tox
-pytest
-pytest-xdist
-pytest-cov
-pytest-benchmark
-Cython
-twine
+setuptools==28.8.0
+setuptools-scm==5.0.2
+pip==20.3.4
+virtualenv==20.13.1
+tox==3.24.5
+pytest==6.1.2
+pytest-xdist==2.2.1
+pytest-cov==2.12.1
+pytest-benchmark==3.4.1
+Cython==0.29.27
+twine==1.15.0
```

Comment:
for reference, this is what the locked requirements in 1.1-maint used previously:
```
setuptools==50.3.0
setuptools-scm==4.1.2
pip==20.2.3
virtualenv==20.0.33
importlib-metadata==1.7.0
pluggy==0.13.1
tox==3.20.0
pytest==6.1.1
pytest-xdist==2.1.0
pytest-cov==2.10.1
pytest-benchmark==3.2.3
Cython==0.29.23
twine==1.15.0
```

Reply:
Sorry. setuptools==28.8.0 is a result of some strange interaction between conda and venv.

Reply:
Cython==0.29.27 is the latest Cython (unless you count pre-releases). pytest==6.1.2 is the last pytest supporting python3.5 (see https://docs.pytest.org/en/stable/changelog.html#id97)

Reply:
to fix the current issue maybe keep these pluggy and importlib-metadata lines at that place.